contributing/documentation/content_guidelines.rst

```diff
@@ -0,0 +1,164 @@
+:banner: banners/contributing.png
+
+==================
+Content Guidelines
+==================
+
+To give the community the best documentation possible, we listed here a few guidelines, tips and
+tricks that will make your content shine at its brightest! While we encourage you to adopt your own
+writing style, some rules still apply to give the reader more clarity and comprehension.
+
+.. note::
+   We strongly recommend contributors to carefully read the other documents in this *Contribution*
+   section of the documentation. Good knowledge of the ins and outs of **RST writing** is required
+   to write and submit your contribution. Note that it also affects your writing style itself.
+
+   - :doc:`introduction_guide`
+   - :doc:`rst_cheat_sheet`
+   - :doc:`rst_guidelines`
+
+Writing style
+=============
```

Comment:
```suggestion
.. _contributing/writing-style:

Writing style
=============
```
And so forth. It makes referencing specific sections easier and much more resilient.